CMakeLists.txt

```diff
@@ -92,27 +108,19 @@ if (CYGWIN)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-exceptions -fno-rtti")
 endif()
 
+if (CLANG_CL)
+    # Since the "secure" replacements that MSVC suggests are not portable, disable
+    # the deprecation warnings. Also disable warnings about use of POSIX functions (i.e. "unlink").
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE")
```

Comment:
What does CRT_SECURE_NO_WARNINGS do exactly?

Reply:
The MSVC stdlib deprecates a bunch of C functions (such as strcpy) and promotes it's own "secure" replacements (strcpy_s). Since filament is supposed to be portable, strcpy_s can't be used, so the warnings aren't very useful.
Details here: https://docs.microsoft.com/en-us/cpp/c-runtime-library/security-features-in-the-crt

Reply:
Good to know, thanks! Sounds like we should take this flag.